chpl/chpl-service/src/main/java/gov/healthit/chpl/search/domain/ListingSearchResponse.java

```diff
@@ -0,0 +1,24 @@
+package gov.healthit.chpl.search.domain;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@XmlRootElement(name = "searchResults")
+@Data
```

Comment:
After the conversation we had about the `@Data` annotation on another PR recently, are we going to stop using them? Or not? Or something else?

Reply:
AFAIK - we should stop using `@Data` on `@Entity` objects only.